### PR TITLE
Add documentation about access and permission

### PR DESCRIPTION
**Pull request by christmex**
Hi,

I make this PR because I think it's kinda hard to find out how can we prevent users before they access the page, then I think it's a good idea to add documentation about the access and permission to every Operation.

Currently, I'm working on a new project and using the latest version of backpack, and I found that when I use
```php
CRUD::denyAccess('list');
```
and not adding this code right after ``` CRUD::denyAccess('list'); ```
```php
abort(403, 'You dont have permission to access this page'); 
```

it will show you this error
image

but when I use this full code
```php
CRUD::denyAccess('list');
abort(403, 'You dont have permission to access this page'); 
```
It works properly, idk if I miss something, let me know if I did something wrong, thank you

also if you agree with this idea, I will continue to add this **access and permission section** to every Operation that backpack provides

Cheers

### CODE REVIEW

**Comment by christmex** [comment]
Hi, 
I think this is what did I missed, @tabacitu already make it clear [here ](https://github.com/Laravel-Backpack/community-forum/discussions/571#discussioncomment-6446879)

I just read that again and realize what did I miss, sorry for that.
About the access and permission documentation, I think we still need it.

**Comment by tabacitu** [comment]
I agree this is something we should document. I will ask @karandatwani92 to rephrase your words here and add it to our docs. Thanks @christmex

**Comment by pxpm** [comment]
Hey @christmex 

That's a bug in backpack v6. So ... thanks for the report? 🙏 🙃 

Doing `CRUD::denyAccess($operations)` should be enough to trigger the 403 forbidden access. 

image

Fixed in https://github.com/Laravel-Backpack/CRUD/pull/5237

**Comment by christmex** [comment]
Hi @tabacitu @pxpm thank you so much 🙏 also if there is something I can help with regarding the access and permission documentation, let me know 🙏, again thank you guys.

Cheers

File: 6.x/crud-operation-list-entries.md [hunk 1]
```diff
@@ -387,3 +387,30 @@ If an error is thrown during the AJAX request, Backpack will show that error in
...
+{
+    //sometimes when we install the backpack permission package we can use something like backpack_user()->can('some_permission')
+    $somePermission = true; 
+    if($somePermission){
+        $this->crud->denyAccess('list');
+        // here's the message for the user, you can change it with whatever you want
+        abort(403, 'You dont have permission to access this page'); 
+    }
+}
+```
```
Comment (promatik) [inline 6.x/crud-operation-list-entries.md:416]:
```suggestion
```

**Review by promatik — APPROVED** [review]

**Comment by promatik** [comment]
Thank you for the report @christmex!
I'll merge this one since it fixes a typo also 👌

**Pull request merged**